### PR TITLE
fixing OCD-1566 and OCD-1970

### PR DESCRIPTION
**Pull request by palakpatel311**
Fixed failed steps from automation test suite

* Moved 3 UI features to UI folder

### CODE REVIEW

File: src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ListingUploadConfirmSteps.java [hunk 1]
```diff
@@ -30,6 +30,7 @@
 import cucumber.api.java.en.And;
 import cucumber.api.java.en.Then;
 import cucumber.api.java.en.When;
+import gov.healthit.chpl.aqa.pageObjects.ChplDownloadPage;
```
Comment (ashwinimore) [inline src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ListingUploadConfirmSteps.java]: Is this import needed here? I don't think any of the definitions from download page are being used in upload-confirm workflow

**Review by ashwinimore — CHANGES_REQUESTED** [review]

File: src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ListingUploadConfirmSteps.java (hunk 1 above)
Comment (palakpatel311) [reply src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ListingUploadConfirmSteps.java]: Updated.

**Review by ashwinimore — APPROVED** [review]

**Review by andlar — CHANGES_REQUESTED** [review]
Need checkstyle fixes. Putting this here as a reminder. Will review more fully when those have been done

**Comment by palakpatel311** [comment]
@andlar  Ashwini has PR open for checkstyle changes. so i am not doing as part of this

**Review by andlar — APPROVED** [review]

**Pull request merged**